Automattic/jetpack
# Jetpack: Deprecate Errors class

Fixes https://github.com/Automattic/vulcan/issues/333


## Proposed changes:

* Deprecated catch_errors() method in Errors class which effectively deprecates the class itself.
* Deprecated catch_errors_on_shutdown() since 'php_errors' state wasn't used anywhere.
*  Removed the code calling the method (modules activation).

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Load Jetpack and see if enabling or disabling modules still works.
* Try breaking a module (eg. Carousel) and toggling it on using the settings page. You should see the error notification pop up but Jetpack should still work. The error will be logged to debug.log depending on wp-config.settings.
